### DISCUSSION TITLE
WASM Deploy Issue

### DISCUSSION THREAD

**Discussion by michalss** [unoplatform/uno#10966] [General]
HI,

Why i cannot properly do publish as framework dependent  anymore. It was possible with Net5 but i cannot do it now. I need to somehow use dotnet serve with linux but ho idea how to publish it ? Option is not there.. And also there is not any dist folder anymore as it use to be.   :(  Can please help me out?

![image](https://user-images.githubusercontent.com/1000729/212114707-9f34274a-ed67-4437-82b5-f00a3a6d8dd4.png)

Thx

fyi @jeromelaban

**Comment by jeromelaban** [comment]
This looks like a visual studio issue, but you can generate publishable package using `dotnet publish -c Release` using the command line.
